cloudify_docker_image/centos_mtu_fix.sh

```diff
@@ -0,0 +1,11 @@
+echo "Enter docker.service file path:"
```

Comment:
Why doesn't this have a hashbang?

Also, please rename all of these scripts to not have the .sh- the language used within them is an implementation detail, no need to make it part of the interface (this also means that they can be re-engineered in python or anything else if we ever want to).

Reply:
Just forgot to put one, now it does.